tensorboard/plugins/pr_curve/summary.py

```diff
@@ -164,6 +168,74 @@ def op(
         description,
         collections)
 
+def pb(tag,
+       labels,
+       predictions,
+       num_thresholds=None,
+       weights=None,
+       display_name=None,
+       description=None):
+  """Creates a PR curves summary protobuf
+
+  Arguments:
+    tag: A name for the generated node. Will also serve as a series name in
```

Comment:
Looks like other summaries call this `name` instead of `tag`. Can we stay consistent with them?

Reply:
I renamed the `tag` parameter to `name` for summaries.